pkg/apis/eventsources/v1alpha1/types.go

```diff
@@ -422,6 +424,25 @@ type StripeEventSource struct {
 	EventFilter []string `json:"eventFilter,omitempty" protobuf:"bytes,5,rep,name=eventFilter"`
 }
 
+// PulsarEventSource refers to event source for Apache Pulsar related events
+type PulsarEventSource struct {
```

Comment:
Let's add a field called `JSONBody` to let users control the type of message body, similar to https://github.com/argoproj/argo-events/blob/master/pkg/apis/eventsources/v1alpha1/types.go#L182